File: 3005. Count Elements With Maximum Frequency/Solution.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maxFrequencyElements = function(nums) {
    dt = {}
    nums.forEach(num=>{
        dt[num] = (dt[num] | 0) + 1;
    })
    let max_val = Math.max(...Object.values(dt));
    let count = 0;
    for(let key in dt){
        if(dt[key] == max_val){
            count += dt[key];
        }
    }
    return count;
};

File: 3005. Count Elements With Maximum Frequency/Solution.py
 class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        dt = Counter(nums)
        max_val = max(dt.values())
        count = 0
        for x,y in dt.items():
            if y==max_val:
                count+=y
        return count
